content-src/components/Hint/Hint.scss

```diff
@@ -0,0 +1,90 @@
+.hint-container {
+  display: inline-block;
+  position: relative;
+  z-index: $hint-z-index;
```

Comment:
Why put the z-index and positioning on the hint-container?  The prompt doesn't need that stuff, and I'm guessing it's really only the explanation that wants it.  If there is a good reason for this, I propose a comment...